packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/media/MediaRecorderBuilder.java

```diff
@@ -85,23 +120,32 @@ public MediaRecorder build() throws IOException, NullPointerException, IndexOutO
       mediaRecorder.setOutputFormat(encoderProfiles.getRecommendedFileFormat());
       if (enableAudio) {
         mediaRecorder.setAudioEncoder(audioProfile.getCodec());
-        mediaRecorder.setAudioEncodingBitRate(audioProfile.getBitrate());
+        mediaRecorder.setAudioEncodingBitRate(
+            (null != audioBitrate && 0 < audioBitrate) ? audioBitrate : audioProfile.getBitrate());
         mediaRecorder.setAudioSamplingRate(audioProfile.getSampleRate());
       }
       mediaRecorder.setVideoEncoder(videoProfile.getCodec());
-      mediaRecorder.setVideoEncodingBitRate(videoProfile.getBitrate());
-      mediaRecorder.setVideoFrameRate(videoProfile.getFrameRate());
+      mediaRecorder.setVideoEncodingBitRate(
+          (null != videoBitrate && 0 < videoBitrate) ? videoBitrate : videoProfile.getBitrate());
```

Comment:
Nit: please reverse comparisons throughout the changes in this file.

Reply:
reversed everywhere